src/main/java/de/tum/cit/aet/artemis/exercise/web/AbstractSubmissionResource.java

```diff
@@ -82,7 +82,7 @@ protected ResponseEntity<List<Submission>> getAllSubmissions(Long exerciseId, bo
                 submission.getParticipation().setExercise(null);
             }
             // Important for exercises with Athena results
-            if (assessedByTutor) {
+            if (assessedByTutor && !exercise.isExamExercise()) {
```

Comment:
Exams should not have Athena results to begin with. We added this condition because the getNonAthenaResults method here also excludes null results, but the exam assessment uses null results to index the correction rounds (I assume to hide who assessed the first round).